src/main/java/org/takes/facets/auth/social/PsFacebook.java

```diff
@@ -91,22 +91,22 @@
     /**
      * Request for fetching app token.
      */
-    private final transient com.jcabi.http.Request request;
+    private final com.jcabi.http.Request request;
 
     /**
      * Facebook login request handler.
      */
-    private final transient WebRequestor requestor;
+    private final WebRequestor requestor;
```

Comment:
@dalifreire are you sure that these new fields can go to equals? Do we have any tests for it?